_components/search-bar.md

```diff
@@ -4,9 +4,59 @@ type: component
 title: Search Bar
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Search_Bar_UI_v1.png">
+<div class="preview preview-search-bar">
+
+  <div class="usa-grid-box">
+    <div class="usa-width-one-half">
+      <form action="#" method="post" class="usa-search usa-search-big">           
+        <fieldset>
+          <legend class="usa-sr-only">Search</legend>
```

Comment:
sorry, that stands for screen reader. So the legend will be viewable to only screen readers and off screen for everyone else.